### PR TITLE
tools: Make static code tests work if pyflakes not present

### PR DESCRIPTION
**Pull request by stefwalter**
The static code tests were skipping all the tests if pyflakes
was not present and installed.

In addition they did not output proper Test Anything Protocol
output. Non TAP output should go to stderr, and we should be
prenting 'not ok' entries if stuff fails

### CODE REVIEW

**Comment by stefwalter** [comment]
```
$ tools/test-static-code 
1..3
ok 1 pyflakes pkg tools
ok 2 pyflakes bots test
ok 3 js-translatable-strings
```

```
$ git diff pkg/
diff --git a/pkg/docker/cockpit-atomic-storage b/pkg/docker/cockpit-atomic-storage
index ae0b618a7..ae593ef21 100755
--- a/pkg/docker/cockpit-atomic-storage
+++ b/pkg/docker/cockpit-atomic-storage
@@ -81,6 +81,7 @@ import json
 import re
 import select
 import errno
+import time
 
 ## Utils
$ tools/test-static-code 
1..3
pkg/docker/cockpit-atomic-storage:84: 'time' imported but unused
not ok pyflakes pkg tools
ok 2 pyflakes bots test
ok 3 js-translatable-strings
$ tools/test-static-code 2> /dev/null
1..3
not ok pyflakes pkg tools
ok 2 pyflakes bots test
ok 3 js-translatable-strings
```

```
$ git diff
diff --git a/test/verify/check-accounts b/test/verify/check-accounts
index ff808f5f9..00149d3ea 100755
--- a/test/verify/check-accounts
+++ b/test/verify/check-accounts
@@ -19,6 +19,7 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 import datetime
+import time
 
 import parent
 from testlib import *
$ tools/test-static-code 
1..3
ok 1 pyflakes pkg tools
test/verify/check-accounts:22: 'time' imported but unused
not ok 2 pyflakes bots test
ok 3 js-translatable-strings
$ tools/test-static-code 2> /dev/null
1..3
ok 1 pyflakes pkg tools
not ok 2 pyflakes bots test
ok 3 js-translatable-strings
```

```
$ git diff pkg/
diff --git a/pkg/users/local.js b/pkg/users/local.js
index 42349e081..013976f9a 100755
--- a/pkg/users/local.js
+++ b/pkg/users/local.js
@@ -38,7 +38,7 @@ function update_accounts_privileged() {
 
     $(".accounts-privileged:not('.accounts-current-account')").update_privileged(
         permission, cockpit.format(
-            _("The user <b>$0</b> is not permitted to modify accounts"),
+            _('The user <b>$0</b> is not permitted to modify accounts'),
             permission.user ? permission.user.name : '')
     );
     $(".accounts-privileged").find("input")
$ tools/test-static-code
1..3
ok 1 pyflakes pkg tools
ok 2 pyflakes bots test
pkg/users/local.js:            _('The user <b>$0</b> is not permitted to modify accounts'),
not ok 3 js-translatable-strings
[stef@falcon cockpit]$ tools/test-static-code 2> /dev/null
1..3
ok 1 pyflakes pkg tools
ok 2 pyflakes bots test
not ok 3 js-translatable-strings
```

```
$ sudo mv /usr/bin/pyflakes /usr/bin/pyflakes-bak
$ tools/test-static-code 
1..3
ok 1 pyflakes pkg tools # SKIP pyflakes not installed
ok 2 pyflakes bots test # SKIP pyflakes not installed
ok 3 js-translatable-strings
```

**Comment by martinpitt** [comment]
Effective diff: https://paste.fedoraproject.org/paste/CrlHITdRew~7easjupMYIw/ (not visible here due to renaming)

File: tools/test-static-code [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    else
+        echo "ok 2 pyflakes bots test"
+    fi
+fi
+
+#
+# wrongly marked translatable strings
+#
+
+if out=$(find src/ pkg/ -name '*.js' -o -name '*.jsx' -o -name '*.es6' | xargs grep "_('"); then
```
Comment (martinpitt) [inline tools/test-static-code]: Any particular reason you dropped the `echo 'ERROR: translatable strings must be marked with _("")'` bit here? (it should go to `>&2` now, of course). Most of us should know what it means, but I'd find it helpful for external contributors to keep the explanation?

Comment (stefwalter) [inline tools/test-static-code]: Yeah we can put that back

Comment (stefwalter) [inline tools/test-static-code]: Added back.

**Pull request merged**